payjoin/src/receive/v2/mod.rs

```diff
@@ -708,7 +708,8 @@ pub mod test {
     #[test]
     fn receiver_ser_de_roundtrip() -> Result<(), serde_json::Error> {
         let session = Receiver { context: SHARED_CONTEXT.clone() };
-        assert_eq!(session.key().as_ref(), session.key().0.as_bytes());
+        let short_id = id(&session.context.s);
+        assert_eq!(session.key().as_ref(), short_id.as_bytes());
```

Comment:
More of a question than a req for change. Was this diff related to the change above on L644? If not generally the pattern in this repo is seperate these out into two seperate commits

Reply:
I recognize the "and" in the commit make it seem they are separate, they are related